KERNEL_CONNECTION.md

```diff
@@ -0,0 +1,128 @@
+# Custom kernel connection example (inside Docker)
+
+**Note**: Hydrogen now supports using kernel gateways ([see the instructions in the main page](https://github.com/nteract/hydrogen/blob/master/README.md)). Using that option might be simpler and will allow you to use the functionality in Windows or Mac very easily.
```

Comment:
Good point. Which is like totally the contrary...

Maybe like this:

---

**Note**: Hydrogen now supports using kernel gateways ([see the instructions in the main page](https://github.com/nteract/hydrogen/blob/master/README.md)). Using that option might be simpler in most of the use cases. For example, it will allow you to use the functionality from a Docker virtual machine in Windows or Mac as easily as if it was Linux.

---

What do you think @lgeiger ?